front/prompt/global_agent_helper_prompt.md

```diff
@@ -76,19 +76,20 @@ To synchronize Slack channels with Dust:
 
 - Inside `Admin` > `Connections` > `Slack`;
 - Navigate to the Slack data source settings;
-- Select the Slack channels you want to synchronize with Dust;
+- Select the Slack channels you want to synchronize with Dust- Dust doesn't synchronize private channels;
 - The @Dust bot will automatically join the selected channels.
 
 THEN, in the Dust app, while signed in as an admin:
 
-- Confirm that you want to index the channels
+- Confirm that you want to synchronize the channels
 
 Once that's done, everyone in the Slack workspace can use the @dust Slackbot. It only has to be installed once.
 
 If an admin has installed Dust, @dust Slackbot will show up as a bot user. Users can chat with @dust by:
 
 - Mentioning @dust in a public channel.
-- @dust doesn't work in private channels.
+- Mentioning @dust in private channels.
```

Comment:
Slack calls that DMs and group DMs. I would write it like that with a link

- Use dust in Direct Messages or group DMs -- you can do that by [turning the group into a private channel](https://slack.com/help/articles/217555437-Convert-a-group-direct-message-to-a-private-channel)

Reply:
sorry this comment is for the line just below

Reply:
LGTM!